internal/generate/flags.go

```diff
@@ -62,11 +60,15 @@ Flags:
 }
 
 func validateConfig(cfg *config) error {
+	if cfg.api == "" {
+		return fmt.Errorf("api must be provided")
+	}
+
 	switch cfg.language {
 	case "cpp":
 		return errNotImplemented
-	case "csharp":
-		return errNotImplemented
+	case "dotnet":
```

Comment:
We need to modify the help text for this - and remove the "command" part, as that's not currently used as far as I can see (and prevented the flags from being parsed).

Reply:
Done